docs/docs/02-configuration/11-queryable-projections.mdx

```diff
@@ -0,0 +1,73 @@
+import Tabs from '@theme/Tabs';
+
+# IQueryable projections
+
+Mapperly does support `IQueryable<T>` projections:
+
+<!-- do not indent this, it won't work, https://stackoverflow.com/a/67579641/3302887 -->
+
+<Tabs>
+<TabItem value="definition" label="Mapper definition">
+
+```csharp
+[Mapper]
+public static partial class CarMapper
+{
+    // highlight-start
+    public static partial IQueryable<CarDto> ProjectToDto(this IQueryable<Car> q);
+    // highlight-end
+}
+```
+
+</TabItem>
+<TabItem value="usage" label="Usage">
+
+```csharp
+var dtos = await DbContext.Cars
+    .Where(...)
+    // highlight-start
+    .ProjectToDto()
+    // highlight-end
+    .ToListAsync();
+```
+
+</TabItem>
+</Tabs>
+
+This is useful in combination with EntityFramework and other ORM solutions
+exposing `IQueryable<T>` as only fields present in the target need to be loaded.
+For queryable projection mappings, Mapperly inlines all mapping code.
+
+:::info
+
+Since queryable projection mappings use `System.Linq.Expressions.Expression<T>` under the hood,
+such mappings have several limitations:
+
+- Object factories are not applied
+- Constructors with unmatched optional parameters are ignored
+- `ThrowOnPropertyMappingNullMismatch` is ignored
+- User implemented mappings are not supported
+- Enum mappings do not support the `ByName` strategy
+- Reference handling is not supported
+
+:::
+
+## Property configurations
+
+To configure property mappings add partial mapping method definitions with attributes as needed.
+Set these methods to private to hide them from callers.
+
+```csharp
+[Mapper]
+public static partial class CarMapper
+{
+    // highlight-start
+    public static partial IQueryable<CarDto> ProjectToDto(this IQueryable<Car> q);
+    // highlight-end
+
+    // highlight-start
+    [MapProperty(nameof(Car.Manufacturer), nameof(CarDto.Producer)]
```

Comment:
Wouldn't it make more sense to support this on the projection method?

Reply:
https://www.itrockt.ch/mitglieder/riok-gmbh/